File: 프로그래머스/lv1/12921. 소수 찾기/소수 찾기.py
def solution(n):
    answer = 1 #2
    
    for i in range(3,n+1,2):
        thtn=True
        for j in range(3,int(i**0.5)+1):
            if i%j==0:
                thtn=False
                break
        if thtn:
            answer+=1
    
    
    return answer

File: 프로그래머스/1/12921. 소수 찾기/소수 찾기.java
class Solution {
    public int solution(int n) {
        int answer = 1; //2 is prime
        
        for(int i=3;i<=n;i+=2){
            boolean b=true;
            for(int j=3;j<=Math.sqrt(i);j++){
                if(i%j==0) {
                    b=false;
                    break;
            }
        }
            if(b) answer++;
        }
        return answer;
    
}
}